config/constants.yml

```diff
@@ -10,11 +10,15 @@
 TERMINUS_VERSION:       '0.12.0'
 
 # Connectivity
-TERMINUS_HOST:          'dashboard.pantheon.io'
+TERMINUS_HOST:          'terminus.pantheon.io'
 TERMINUS_PORT:          443
 TERMINUS_PROTOCOL:      'https'
 TERMINUS_SSH_HOST:      null
 
+# User Prompts
+DASHBOARD_HOST:         'dashboard.pantheon.io'
```

Comment:
@bensheldon OK. Cool.  I think it's just  `site dashboard` and instructions for machine token generation.